src/H5Tprivate.h

```diff
@@ -41,11 +41,13 @@ typedef struct H5T_t H5T_t;
 #define H5T_GET_SHARED(T)           ((T)->shared)
 #define H5T_GET_MEMBER_OFFSET(T, I) ((T)->u.compnd.memb[I].offset)
 #define H5T_GET_MEMBER_SIZE(T, I)   ((T)->u.compnd.memb[I].shared->size)
+#define H5T_GET_FORCE_CONV(T)       ((T)>shared->force_conv)
```

Comment:
```suggestion
#define H5T_GET_FORCE_CONV(T)       ((T)->shared->force_conv)
```

Reply:
That only exists in this PR, until it is merged it is not in 1.12. Neal needs to make the change for it to appear in this PR. 
OR
Merge this PR then go back and edit.